src/main/java/emu/grasscutter/Config.java

```diff
@@ -74,7 +74,7 @@ public static class GameServerOptions {
 		public int InventoryLimitMaterial = 2000;
 		public int InventoryLimitFurniture = 2000;
 		public int InventoryLimitAll = 30000;
-		public int MaxAvatarsInTeam = 4;
+		public int MaxAvatarsInTeam = 9;
```

Comment:
Please keep this the default value of 4.

Reply:
okay，changed